authapi/models.py

```diff
@@ -9,14 +9,24 @@ class SeedOrganization(models.Model):
     modified_at = models.DateTimeField(auto_now=True)
     archived = models.BooleanField(default=False)
 
+    def get_active_teams(self):
+        return self.seedteam_set.filter(archived=False)
+
 
 class SeedPermission(models.Model):
-    # TODO: Implement SeedPermission model
-    pass
+    permission_type = models.CharField(max_length=128)
```

Comment:
I thought that I couldn't, but I just tried, and it turns out I can. So I'll change it